sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/memory.scala

```diff
@@ -148,7 +167,7 @@ case class MemoryStream[A : Encoder](id: Int, sqlContext: SQLContext)
     }
   }
 
-  private def generateDebugString(
+  protected def generateDebugString(
```

Comment:
why this change?

Reply:
honestly not sure why this ended up happening - un-changed it